libavformat/naxat_asd.c

```diff
@@ -217,7 +217,7 @@ static int naxat_asd_bnk_read_header(AVFormatContext *s)
         st->index = n;
     }
 
-    avio_seek(pb, header_end, SEEK_SET);
+    avio_seek(pb, s->streams[0]->priv_data->start_offset, SEEK_SET);
```

Comment:
You need to cast to priv_data proper type:

+        if (n == 0) {
| +            NaxatASDStream *ast = st->priv_data;
| +
| +            header_end = ast->start_offset;